docs/architecture.md

```diff
@@ -88,6 +88,7 @@ Actually `OMG.API.EthereumEventListener` setup with `:slow_validator`.
 - used only in Watcher
 - feeds `OMG.API.State` with valid exits and challenges invalid exits (caused by root chain reorgs)
```

Comment:
needs updating so that the descriptions of slow and fast validators are consistent (for example "challenges invalid exits" is inconsistent)

Reply:
`### :fast_validator

Actually `OMG.API.EthereumEventListener` setup with `:fast_validator`.

- used only in Watcher
- validates exits and pushes them to `WatcherDB`
- emits byzantine events to `OMG.Watcher.Eventer`

### :slow_validator

Actually `OMG.API.EthereumEventListener` setup with `:slow_validator`.

- used only in Watcher
- validates exits and pushes them to `OMG.API.State`
- challenges exits
- emits byzantine events to `OMG.Watcher.Eventer``